### PR TITLE
Run `npm ci` when a lockfile is available

### LINKED ISSUE #8: Run `npm ci` when a lockfile present
At the moment this always runs `npm install` - `npm ci` performs a clean install, is faster and the installs are essentially frozen (i.e. better reproducibility).

### PR DESCRIPTION
**Pull request by dominykas**
Closes #8.

Doing this as a separate step, because I'd like to keep the default shell (Powershell on Windows) for the actual install step (I suspect we'll have to make it configurable, but that's out of scope here - will wait for someone to ask for that).

Test runs:

- shrinkwrap: https://github.com/dominykas/pkgjs-action/actions/runs/1647166714
- package-lock: https://github.com/dominykas/pkgjs-action/actions/runs/1647168303
- no lockfile: https://github.com/dominykas/pkgjs-action/actions/runs/1647171945

### CODE REVIEW

File: .github/workflows/node-test.yaml [hunk 1]
```diff
@@ -116,8 +116,19 @@ jobs:
...
 
+    - name: Prepare install script
+      id: prepare-install-script
+      shell: bash
+      run: |
+        if [ -f package-lock.json ] || [ -f npm-shrinkwrap.json ]
+        then
+          echo "::set-output name=install-command::npm ci"
+        else
+          echo "::set-output name=install-command::npm install"
```
Comment (Eomm) [inline .github/workflows/node-test.yaml:127]: What about yarn.lock?

Comment (dominykas) [reply .github/workflows/node-test.yaml:127]: `yarn` support not yet implemented - #6 - it needs a little bit more thought.

Comment (ljharb) [reply .github/workflows/node-test.yaml:127]: `npm install` and `npm ci` both read `yarn.lock` automatically in npm 7+.

Comment (dominykas) [reply .github/workflows/node-test.yaml:127]: You think we should run `npm ci` when `yarn.lock` is present (until #6 is done)?

Comment (ljharb) [reply .github/workflows/node-test.yaml:127]: Yes, that would make the most sense.

Comment (dominykas) [reply .github/workflows/node-test.yaml:127]: https://github.com/pkgjs/action/pull/37/commits/4b7e44d3e150f8299e5f14e88d0fa0e18e74121c

**Review by ljharb — APPROVED** [review]

**Review by Eomm — APPROVED** [review]

**Pull request merged**